packages/tpetra/core/test/ImportExport2/CMakeLists.txt

```diff
@@ -7,3 +7,12 @@ TRIBITS_ADD_EXECUTABLE_AND_TEST(
   ARGS "--globally-reduce-test-result --output-show-proc-rank --output-to-root-rank-only=-1"
   STANDARD_PASS_OUTPUT
   )
+
+TRIBITS_ADD_EXECUTABLE_AND_TEST(
+  ImportExport2_CrsSortingUtils
+  SOURCES
+    ImportExport2_CrsSortingUtils.cpp
+    ${TEUCHOS_STD_UNIT_TEST_MAIN}
```

Comment:
We talked over the phone today about using a different unit test `main()` function.  For an example in Tpetra that puts the `main()` function in the unit test file, see `tpetra/core/test/HashTable/computeOffsetsFromCounts.cpp`.

Reply:
Thanks for the link!  I'll look in to the alternative `main()`